packages/trader/src/AppV2/Components/Guide/Description/video-preview.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Stream } from '@cloudflare/stream-react';
+import { CaptionText } from '@deriv-com/quill-ui';
+import { LabelPairedPlayMdFillIcon } from '@deriv/quill-icons';
+import { Localize, localize } from '@deriv/translations';
+import { CONTRACT_LIST } from 'AppV2/Utils/trade-types-utils';
+
+type TVideoPreview = {
+    contract_type: string;
+    toggleVideoPlayer: () => void;
+    video_src: string;
+};
+
+const VideoPreview = ({ contract_type, toggleVideoPlayer, video_src }: TVideoPreview) => {
+    if (contract_type === CONTRACT_LIST.TURBOS) return null;
+
+    return (
+        <div className='guide-video__wrapper'>
+            <div
+                className='guide-video__preview'
+                data-testid='dt_video_preview'
+                onClick={toggleVideoPlayer}
+                onKeyDown={toggleVideoPlayer}
+            >
+                <Stream
+                    autoplay={false}
+                    controls={false}
```

Comment:
@kate-deriv could you please check if `autoplay` and `controls` a false by default in `Stream` same as in `video` tag? in this case, they can be omitted.

Reply:
Sure, let me check

Reply:
For `video` tag: we should pass `autoplay` otherwise it's not playing.
For `Stream` : `autoplay={false}` and `controls={false}` can be omitted.